### PR TITLE
feature/NUI-5064-Fix-Stepper-CSS-for-horizontal-wizard

### PR DESCRIPTION
**Pull request by Vita-Karas**

### CODE REVIEW

**Review by DmytroPutrin — APPROVED** [review]

File: packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less [hunk 1]
```diff
@@ -1,63 +1,53 @@
...
-        }
+    &--disabled {
+        pointer-events: none;
+        cursor: default;
+    }
 
-        .nui-step-header-ripple {
-            pointer-events: none;
-        }
+    .nui-step-header__ripple {
```
Comment (dmytrofirsov) [inline packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less:23]: Maybe we may want to continue with the `nui-wizard-step-header` naming here as well, just like you did for others above.  What is the purpose of this class by the way? Why do we need it?

File: packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less [hunk 2]
```diff
@@ -1,63 +1,53 @@
...
-            // TODO: switch back to mixin .focus-outline() when z-index is removed in the scope of NUI-6103
-            outline: none;            
-            box-shadow: inset 0 0 0 @nui-line-medium fade(@nui-color-selected-contrast, 50%);
-            border-radius: @nui-radius-default;
-        }
+    &:focus {
+        // TODO: switch back to mixin .focus-outline() when z-index is removed in the scope of NUI-6103
+        outline: none;
+        box-shadow: inset 0 0 0 @nui-line-medium
+            fade(@nui-color-selected-contrast, 50%);
```
Comment (dmytrofirsov) [inline packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less:31]: maybe keeping in inline would look better? what do you think?

File: packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less [hunk 3]
```diff
@@ -1,63 +1,53 @@
...
+    &:focus:not(:focus-visible) {
+        box-shadow: unset;
+        border-radius: unset;
     }
 
     &-icon {
-        border-radius: 50%;
-        height: @wizard-label-header-height;
-        width: @wizard-label-header-height;
-        flex-shrink: 0;
```
Comment (dmytrofirsov) [inline packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less:52]: Some styles were removed here? Was this discussed with ux?

**Comment by dmytrofirsov** [comment]
A very good job overall @Vita-Karas!

File: packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less (hunk 1 above)
Comment (Vita-Karas) [reply packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less:23]: This redundant class removed

File: packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less (hunk 2 above)
Comment (Vita-Karas) [reply packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less:31]: Will fix, thanks

File: packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less (hunk 3 above)
Comment (Vita-Karas) [reply packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less:52]: No, this wasn't discussed with ux. These styles are unnecessary. According to the mockup, we need just width and height for the icon. https://marvelapp.com/prototype/5d7h3hd/screen/46370577/

**Review by dmytrofirsov — APPROVED** [review]

**Review by ivankozlovskyi — APPROVED** [review]

**Pull request merged**